toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -854,29 +853,40 @@ func hydrateFiles(fileTypeToHydrate fileType, specFile, workingDir string, srcCo
 		}
 	}
 
+	missingFiles := []string{}
 	for fileNeeded, alreadyHydrated := range fileHydrationState {
 		if !alreadyHydrated {
-			err = fmt.Errorf("unable to hydrate file: %s", fileNeeded)
-			logger.Log.Error(err)
+			missingFiles = append(missingFiles, fileNeeded)
+			logger.Log.Errorf("Unable to hydrate file: %s.", fileNeeded)
```

Comment:
nit: could you remove the period from this message? otherwise the filename might appear to end with it

Reply:
Ah, I have not though about that. Will do!